man/foreign.doc

```diff
@@ -3164,7 +3166,14 @@ print_message/2 through the C interface.
 
 \begin{description}
     \cfunction{int}{PL_raise_exception}{term_t exception}
-Generate an exception (as throw/1) and return \const{FALSE}. This
+Generate an exception (as throw/1) and return \const{FALSE}.
```

Comment:
I don't like this level of detail.   Preserving exceptions is a difficult topic that has changed several times over the years.   Besides, this only covers the exception `term_t`.    The underlying term also needs protection.